src/config.ts

```diff
@@ -238,7 +238,7 @@ const userPlaceColorsArray: [string, Color][] = [
     ["purple", purple],
     ["indigo", indigo],
     ["cyan", cyan],
-    ["deepPurple", deepPurple],
+    ["purple", purple],
```

Comment:
Please note, that purple is already used in the list. Please choose a different color from this list https://www.w3.org/wiki/CSS/Properties/color/keywords which is not used yet.